core/vmpu/inc/vmpu.h

```diff
@@ -205,6 +205,9 @@ extern int vmpu_is_region_size_valid(uint32_t size);
  */
 extern uint32_t vmpu_round_up_region(uint32_t addr, uint32_t size);
 
+/** Invalid box id for use in marking objects with invalid ownership. */
+#define UVISOR_BOX_ID_INVALID ((uint8_t) -1)
```

Comment:
If this were in `vmpu_exports.h`, we wouldn't have to redefine it as -1 in the unsupported files.